development.md

```diff
@@ -0,0 +1,24 @@
+### Get  Started with Development 
+#
+
+## prerequisites
+
+- installed go if not installed. Follow the below link for more details
+
+[go installation] (https://go.dev/dl/)
+
+- git should be installed for cloning 
+
+- usage of ide. Any ide can be used but visual studio is preferable 
+
+- For kind docker needs to be installed. go through below link 
+[docker installation] (https://docs.docker.com/engine/install/)
+
+- After installation of docker follow the below link for installation of kind 
+
+[kind installation] (https://kind.sigs.k8s.io/docs/user/quick-start/)
+
+- After all above steps are completed perform the steps mentioned in Readme.md for installation of hydrophone and testing it
```

Comment:
I think it would be best for us to move the steps from the README.md into this file and update the README to point folks here for getting started.

Reply:
moved the steps from readme to development.md and gave pointer to refer development.md